lib/diagnostic.rb

```diff
@@ -71,18 +73,18 @@
 
 # What value will be returned. Write your response as a Ruby string.
 
-Diagnostic.flow_control = 'your response here'
+Diagnostic.flow_control = 'Just your average billionaire'
 
```

Comment:
the answer is 'The Dark Knight' because 'batman' is true since we defined it above